frappe/public/js/frappe/form/multi_select_dialog.js

```diff
@@ -128,7 +98,80 @@ frappe.ui.form.MultiSelectDialog = Class.extend({
 		this.dialog.show();
 	},
 
-	bind_events: function() {
+	make_dialog_header: function () {
+		let fields = [];
+		let me = this;
+		this.col_0 = [
+			{
+				fieldtype: "Data",
+				label: __("Search Term"),
+				fieldname: "search_term"
+			}
+		];
+		this.col_1 = [];
+		this.col_2 = [];
```

Comment:
Why add these values to `this` it's not used anywhere else

Reply:
done